src/pages/ProfilePage.js

```diff
@@ -136,12 +136,9 @@ function ProfilePage(props) {
     const shouldShowNotificationPreference = !_.isEmpty(props.report) && props.report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN;
     const notificationPreference = shouldShowNotificationPreference ? props.translate(`notificationPreferencesPage.notificationPreferences.${props.report.notificationPreference}`) : '';
 
-    // eslint-disable-next-line rulesdir/prefer-early-return
     useEffect(() => {
-        if (ValidationUtils.isValidAccountRoute(accountID) && !hasMinimumDetails) {
```

Comment:
What are your thought on removing the `isValidAccountRoute(accountID)` check? I know we said "unconditionally" multiple times, but I'm not sure how literal was that. I thought it was related to `hasMinimumDetails` check in practice.

Do you know what account routes are "invalid"?

Reply:
Here is the new PR
https://github.com/Expensify/App/pull/37223

Reply:
CLA signing is unrelated to Git GPG commit signing. The only common part is the word "sign". CLA is an agreement that you accept as a human being. Git commit signing is a cryptographic proof of your authorship of Git commits.

Anyway, as the CLA check is successful on the second PR, let's move the discussion to https://github.com/Expensify/App/pull/37223

Please close **this** PR!

Reply:
Also, applying changes on the PR is a very standard procedure and, in this project, it's done by adding new commits on top of the old ones. Sometimes people choose commit names as trivial as "Update after PR feedback", but in this case something like "Add the `isValidAccountRoute` check back" would be more suitable.

In other project, force-pushing to create a new "PR revision" could be preferred, but in Expensify force-pushing is disallowed (or at least discouraged) after the first review round. I didn't make this rule.

This is just FYI, as for now, we're moving to https://github.com/Expensify/App/pull/37223.

Reply:
Got it, thanks for explaining. I'll close this PR.